doc/symbol.md

```diff
@@ -34,7 +34,7 @@ If an error occurs, a `Napi::Error` will get thrown. If C++ exceptions are not
 being used, callers should check the result of `Napi::Env::IsExceptionPending` before
 attempting to use the returned value.
 
-### Utf8Value
+### Wellknown
```

Comment:
It shlould be `WellKnown` insteand of `Wellknown`. The `k` should be in uppercase.